File: SoalB/PHP/PC.php
<?php


class PC
{
    /* atribut */
    private $CPU, $RAM, $DISK;
    private $pcPrice;

    /* construct */
    function __construct($CPU, $RAM, $DISK, $pcPrice)
    {
        $this->CPU = $CPU;
        $this->RAM = $RAM;
        $this->DISK = $DISK;
        $this->pcPrice = $pcPrice;
    }

    /* Membuat get set untuk class CPU */
    function setCPU($CPU)
    {
        $this->CPU = $CPU;
    }
    function getCPU()
    {
        return $this->CPU;
    }

    /* Membuat get set untuk class RAM */
    function setRAM($RAM)
    {
        $this->RAM = $RAM;
    }
    function getRAM()
    {
        return $this->RAM;
    }

    /* Membuat get set untuk class DISK */
    function setDISK($DISK)
    {
        $this->DISK = $DISK;
    }
    function getDISK()
    {
        return $this->DISK;
    }

    /* Membuat get set untuk menghitung total harga */
    function setPcPrice($pcPrice)
    {
        $this->pcPrice = $pcPrice;
    }
    function getPcPrice()
    {
        return $this->pcPrice;
    }

    /* menampilkan hasil data PC */
    function printPC()
    {
        $this->CPU->printCPU();
        $this->RAM->printRAM();
        $this->DISK->printDISK();
        echo "Total Price(IDR)      : " . $this->getPcPrice() . "<br/>";
        echo "******************************" . "<br/>";
    }

    /*destructor */
    function __destruct()
    {
    }
}

File: SoalB/Java/PC.java
public class PC {
    /* atribut dan pemanggilan class */
    private CPU cpuA;
    private RAM ramB;
    private DISK diskC;
    private int pcPrice;

    /* constructor */
    public PC(CPU CPUA, RAM RAMB, DISK DISKC, int pcPrice) {
        cpuA = CPUA;
        ramB = RAMB;
        diskC = DISKC;
        this.pcPrice = pcPrice;
    }

    public PC() {

    }

    /* membuat get set method untuk class CPU */
    public void setCPU(CPU CPUA) {
        cpuA = CPUA;
    }

    public CPU getCPU() {
        return cpuA;
    }

    /* membuat get set method untuk class CPU */
    public void setRAM(RAM RAMB) {
        ramB = RAMB;
    }

    public RAM getRAM() {
        return ramB;
    }

    /* membuat get set method untuk class CPU */
    public void setDISK(DISK DISKC) {
        diskC = DISKC;
    }

    public DISK getDISK() {
        return diskC;
    }

    /* membuat get set method untuk total harga PC */
    public void setPricePC(int pcPrice) {
        this.pcPrice = pcPrice;
    }

    public int getPricePC() {
        return pcPrice;
    }

    /* menampilkan data PC */
    public void printPC() {
        cpuA.printCPU();
        ramB.printRAM();
        diskC.printDISK();
        System.out.println("Total Price(IDR)    : " + this.getPricePC());
        System.out.println("######################################");
    }
}
